landingPage.css

```diff
@@ -0,0 +1,153 @@
+body, ul {
+    margin: 0;
+    padding: 0;
+}
+
+/* Navigation Bar */
+header {
+    background-color: #333;
+    color: #fff;
+}
+
+nav {
+    width: 1200px;
+    margin: 0 auto;
+    height: 100px;
+}
+
+nav img {
+    max-width: 8%;
+    height: auto;
+    border: 2px solid black;
+}
+
+nav ul {
+    padding: 50px;
+    margin: 0px;
+    list-style: none;
+    text-align: center;
+    display: inline-block;
+    vertical-align:top;
+}
+
+nav ul li {
+    display: inline;
+    margin: 24px 48px;
+}
+
+nav a {
+    text-decoration: none;
+    color: #fff;
+    font-size: 16px;
+}
+
+nav a:hover {
+    text-decoration: underline;
+}
+
+/* Hero Section */
+.hero {
+    height: 600px;
+    text-align: center;
+}
+
+.hero img {
```

Comment:
I see what you're trying to achieve with img, I think it's better if we render the image as a background so we can easily have text on the image. Then you won't need complex css positioning such as `transform`, `position: absolute` for your texts.

See: https://www.freecodecamp.org/news/css-background-image-with-html-example-code/